tests/ui/test_events.py

```diff
@@ -36,3 +36,32 @@ def on_click_inner(*ignore):
     inner_sel.click()
     page_session.locator("text=Clicked").wait_for()
     mock_outer.assert_called_once()
+
+
+def test_mouse_event(solara_test, page_session: playwright.sync_api.Page):
+    div = vue.Html(tag="div", children=["Click Me!"])
+    last_event_data = None
+
+    def on_click(widget, event, data):
+        nonlocal last_event_data
+        last_event_data = data
+        div.children = ["Clicked"]
+
+    div.on_event("click", on_click)
+    display(div)
+
+    # click in the center of the div
+    div_sel = page_session.locator("text=Click Me!")
+    div_sel.wait_for()
+    box = div_sel.bounding_box()
+    assert box is not None
+    # if we pass floats (like 12.5) we get back ints from the
+    # event (12 in the case of 12.5), so we cast to int
+    cx = int(box["x"] + box["width"] / 2)
```

Comment:
Not important, but for my context; why do we need to click the center? Isn't an offset of top and left of 1px enough?

Reply:
Good point, it seems that not even an offset was needed, just box['x']. My brain was wired to do this due to flakey-tests.

Reply:
I now see in your change that even the offset isn't necessary :-)

Reply:
GitHub is confused :)
<img width="1102" alt="image" src="https://github.com/widgetti/ipyvue/assets/1765949/b7381695-cd8e-4a5d-bf17-497ca584f6cf">

(6 minutes ago is before 5 minutes ago)

Reply:
ah